src/main.d

```diff
@@ -139,6 +139,11 @@ int main(string[] args)
 		log.error(e.msg);
 		log.error("Try 'onedrive -h' for more information");
 		return EXIT_FAILURE;
+	} catch (Exception e) {
+		// error
+		log.error(e.msg);
+		log.error("Try 'onedrive -h' for more information");
+		return EXIT_FAILURE;
 	}
```

Comment:
This is strange - according to the API docs getopt should throw a `GetOptException`, see https://dlang.org/library/std/getopt.html so why is this necessary? But I can confirm that it is necessary ... so in principle I agree ;-)